bench/gateway-without-auth.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const Fastify = require('fastify')
+const mercurius = require('mercurius')
+
+const app = Fastify()
+
+app.register(mercurius, {
+  gateway: {
```

Comment:
There is probably a bit more work to change the default settings, as I think they might be limiting the perf here.

Reply:
sure thing, I'll take a look at the mercurius options to see if I can improve perf